model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -312,12 +312,12 @@ message ProcessBundleProgressRequest {
 // An SDK can report metrics using an identifier that only contains the
```

Comment:
I can't provide a suggestion for the first line, please adjust to `A request to provide full MonitoringInfo for a given id.`